server/src/modules/manage-session/manage-session.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Body, Controller, Get, Param, Post, Put } from '@nestjs/common';
+
+import { ManageSessionService } from './manage-session.service';
+import { ManageSessionDto } from './ManageSession.dto';
+
+@Controller('manage-session')
+export class ManageSessionController {
+  constructor(private msService: ManageSessionService) {}
+
+  @Get()
+  showAllSessions(): Promise<ManageSessionDto[]> {
```

Comment:
Check that swagger doc generates for these endpoints

Reply:
I've added swagger docs to entities